src/main/java/io/confluent/connect/hdfs/DataWriter.java

```diff
@@ -274,7 +274,7 @@ private void configureKerberosAuthentication(Configuration hadoopConfiguration)
           hostname
       );
       UserGroupInformation.loginUserFromKeytab(principal, connectorConfig.connectHdfsKeytab());
-      final UserGroupInformation ugi = UserGroupInformation.getLoginUser();
+      final UserGroupInformation ugi = UserGroupInformation.loginUserFromKeytabAndReturnUGI(principal, connectorConfig.connectHdfsKeytab());
```

Comment:
If the line exceeds 100 characters, mvn checkstyle error occurs, so it would be a good idea to change it like this.

```suggestion
      final UserGroupInformation ugi = UserGroupInformation.loginUserFromKeytabAndReturnUGI(
            principal, 
            connectorConfig.connectHdfsKeytab()
      );
```